systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -61,5 +64,15 @@ private static long timeoutForClientFinishJob(int messagesCount) {
         // need to add at least 1-2minutes for finishing the job
         return (long) messagesCount * 1000 + Duration.ofMinutes(2).toMillis();
     }
+
+    /**
+     * Method which generates random consumer group name
+     * @return consumer group name with pattern: my-consumer-group-*-*
+     */
+    public static String generateRandomConsumerGroup() {
+        int salt = rng.nextInt(Integer.MAX_VALUE);
+
+        return CONSUMER_GROUP_NAME + salt;
+    }
```

Comment:
Please update also `AbstractKafkaClient`. There we are using the same method. So remove it  from `AbstractKafkaClient` and use this one from `ClientUtils` :)

Reply:
Yeah sure! Good point.